src/BarChart/BarSeries/BarSeries.tsx

```diff
@@ -192,7 +192,7 @@ export const BarSeries: FC<Partial<BarSeriesProps>> = ({
 
   const onMouseMove = useCallback((event) => {
     // Manuallly call mouse move so we don't have to kill bar pointer events
-    ref.current?.onMouseMove(event);
+    ref.current?.triggerMouseMove(event);
```

Comment:
Was this just a function rename?

Reply:
There was no function `onMouseMove` within the ref, I added one with the `useImperativeHandle` hook and named it `triggerMouseMove`. It's a wrapper around the `onMouseMove` function within TooltipArea so I gave it a diff name